docs/develop/vscode-for-scripts.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Visual Studio Code for Office Scripts (preview)
+description: How to setup the Office Scripts Code Editor to connect with VS Code on the web.
+ms.date: 11/04/2022
+ms.localizationpriority: medium
+---
+
+# Visual Studio Code for Office Scripts (preview)
+
+[Visual Studio Code on the web](https://vscode.dev/) lets users edit anything from anywhere. Connect your Office Scripts experience to this popular code editor to start scripting outside of the workbook.
+
+:::image type="content" source="../images/vscode-script-editor.png" alt-text="An Excel on the web window with the Code Editor open next to a VS Code on the web window with an open script.":::
+
+Visual Studio Code has a few advantages over the built-in Code Editor.
+
+- Full-screen editing! Your script doesn’t have to share screen space with the workbook any more.
+- Edit multiple scripts at once! Quickly switch between scripts to share code from your other automations.
+- Extensions! Use your favorite VS Code extensions for spell checking, formatting, and whatever else helps you get the job done.
+
+> [!NOTE]
+> This feature is in preview. It's subject to change based on feedback. If you encounter any issues, please report them through the **Feedback** button in Excel. The following are known issues with the current version of the feature.
+>
+> - Visual Studio Code can only be connected to Office Scripts through Excel on the web.
```

Comment:
Do we need to differentiate between VS Code Online and VS Code Desktop?

Reply:
We're guiding partners to say either just "VS Code" or the full "VS Code for the Web" - as long as we're not introducing new terms (like "VS Code Web" or "VS Code on the web") there aren't concerns from our side.